hack/test-stirunimage.sh

```diff
@@ -8,14 +8,14 @@ export PATH="$PWD/_output/local/bin/$(go env GOHOSTOS)/$(go env GOHOSTARCH):$PAT
 
 function time_now()
 {
-  date +%s000
+    date +%s000
```

Comment:
i don't really see an issue w/ doing some formatting cleanup when you're already touching a file.  if we force people to create a separate commit/PR it's just going to make people more reluctant to do it (or forget to do it).  Especially since it increases the risk you're going to create a merge conflict with your other PR.

I'm having a hard time envisioning the scenario where this delta baffles someone, or where someone is going to be digging through our git history looking for the commit that reformatted whitespace.